src/components/CampaignInteractionStepsForm.jsx

```diff
@@ -24,6 +23,7 @@ import GSAutoComplete from "./forms/GSAutoComplete";
 import { makeTree } from "../lib";
 import { dataTest } from "../lib/attributes";
 import withMuiTheme from "../containers/hoc/withMuiTheme";
+import AdminTextPreview from './AdminTextPreview';
```

Comment:
After a lot of experimentation, I realize that this data wasn't coming through in the same that I'd like.  I didn't have access to the texter, the customFields. 

It's fuzzy, but I feel like I remember getting these props and thinking, "where's the rest?"